### PR TITLE
Document how to configure FixedUpdate

### PR DESCRIPTION
**Pull request by stepancheg**

### CODE REVIEW

File: crates/bevy_app/src/main_schedule.rs [hunk 1]
```diff
@@ -71,6 +71,9 @@ pub struct RunFixedUpdateLoop;
 ///
 /// The exclusive `run_fixed_update_schedule` system runs this schedule.
 /// This is run by the [`RunFixedUpdateLoop`] schedule.
+///
+/// Frequency of execution is configured by inserting `Time<Fixed>` resource, 64 Hz by default.
+/// See the [example](https://github.com/bevyengine/bevy/blob/main/examples/time/time.rs).
```
Comment (killercup) [inline crates/bevy_app/src/main_schedule.rs:76]:
```suggestion
/// See [this example](https://github.com/bevyengine/bevy/blob/main/examples/time/time.rs).
```

File: crates/bevy_app/src/main_schedule.rs [hunk 2]
```diff
@@ -71,6 +71,9 @@ pub struct RunFixedUpdateLoop;
 ///
 /// The exclusive `run_fixed_update_schedule` system runs this schedule.
 /// This is run by the [`RunFixedUpdateLoop`] schedule.
+///
+/// Frequency of execution is configured by inserting `Time<Fixed>` resource, 64 Hz by default.
```
Comment (killercup) [inline crates/bevy_app/src/main_schedule.rs:75]: Can we link to the time type with the generic? maybe as two links?

**Review by killercup — APPROVED** [review]
Better than before, two nits :)

**Review by alice-i-cecile — APPROVED** [review]

File: crates/bevy_app/src/main_schedule.rs [hunk 3]
```diff
@@ -71,6 +71,9 @@ pub struct RunFixedUpdateLoop;
...
+/// See [this example](https://github.com/bevyengine/bevy/blob/main/examples/time/time.rs).
```
Comment (alice-i-cecile) [inline crates/bevy_app/src/main_schedule.rs:76]:
```suggestion
/// See [this example](https://github.com/bevyengine/bevy/blob/latest/examples/time/time.rs).
```

Comment (alice-i-cecile) [reply crates/bevy_app/src/main_schedule.rs:76]: Link to latest, not main.

**Comment by alice-i-cecile** [comment]
I'll let this sit until Monday if you wanna add more links <3

File: crates/bevy_app/src/main_schedule.rs (hunk 2 above)
Comment (stepancheg) [reply crates/bevy_app/src/main_schedule.rs:75]: `bevy_app` does not depend on `bevy_time`. We can only link to docs.rs I think?

Comment (killercup) [reply crates/bevy_app/src/main_schedule.rs:75]: Hah, didn't expect that, but makes sense. I don't think we need to link to it then :)

**Review by killercup — APPROVED** [review]

**Pull request merged**